### PR TITLE
Adds Process Flow png and update change of fee md

### PR DESCRIPTION
**Pull request by prtandrup**
## Description

This PR does the following: 
- Adds the Change of Charge Process Flow diagram + drawio file
- Use the flow in the md files; fee -, subscription - and tariff price list. 

Following this PR, another PR will be submitted that updates the architechture diagram.

## References

* AzDO #118636

### CODE REVIEW

File: docs/business-processes/change-of-fee.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ A fee is a price for a single service or task at a given date related to a meter
 Association of fees to metering points are performed using business process _Settlement master data for a metering point –
 subscription, fee and tariff links_.
 
+## Process flow
+
+The following process flow depicts the different paths a change of fee price list can take through the domain.
+It also shows the micro services involved along with the activities they perform.
+
+![Process flow](.././images/ChangeOfChargePriceListProcessFlow.png)
```
Comment (bemwen) [inline docs/business-processes/change-of-fee.md:13]: What tool was made to make it? Is there a source that should be added?

**Review by bemwen — APPROVED** [review]
Nice work.

**Review by BjarkeMeier — COMMENTED** [review]
Looks good. Perhaps we could talk about the structure of the documentation since this diagram was inserted in change-of-fee but applies to all change-of-charge flows?

File: docs/business-processes/change-of-fee.md (hunk 1 above)
Comment (prtandrup) [reply docs/business-processes/change-of-fee.md:13]: I use Drawio, and  the diagram is included in the png file. However, I will, when I believe the flow is final, add the drawio file as well :)

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**